### DISCUSSION TITLE
fa/blog/2024/09/23/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4503] [Content Help کمک به اصلاح محتوا]
# fa/blog/2024/09/23/

Hiddify's Second Anniversary

https://hiddify.com/fa/blog/2024/09/23/

**Comment by NedaMani** [comment]
زنده باد! پایدار باشید🔥❤️

**Comment by umidomm** [comment]
دم شما گرم،امیدوارام انتظار طولانی بابت پنل مرکزی به زودی به سر برسه،با آرزوی موفقیت
